web/src/main/webapp/js/graphhopper/GHRequest.js

```diff
@@ -70,7 +71,7 @@ GHRequest.prototype.init = function (params) {
         else if (val === "true")
             val = true;
 
-        if (key === "point" || key === "mathRandom" || key === "do_zoom" || key === "layer")
+        if (key === "point" || key === "mathRandom" || key === "do_zoom" || key === "layer" || key === "useMiles")
```

Comment:
Ok, I was not sure if I should follow the convention of  "do_zoom" or "mathRandom". I will change the url parameter from "useMiles" to "use_miles", then. Shall I change "mathRandom" to "math_random" for consistency ? What about the java script variables ? In the GHRequest constructor function for example there is a variable `this.do_zoom` and another one `this.dataType` so both underscore and camel case are used next to each other.